explainers/hidden-content-explainer.md

```diff
@@ -200,24 +205,87 @@ representing the active match is collapsed or it has a style which makes it
 invisible, the scroll will be canceled. Otherwise, the target match will be
 scrolled into view.
 
-### Alternatives Considered
+## Accessibility
+
+`content-visibility: hidden-matchable` subtrees will not be included in the
+accessibility tree because they are not visible on the screen. This is the same
+as `content-visibility: hidden` and offscreen/unrendered `content-visibility:
+auto` subtrees.
+
+The content inside `content-visibility: hidden-matchable` subtrees will still
+remain accessible to assisstive technology users because there should always be
+a way to reveal the content besides using find-in-page, such as a button to
+expand the content like a `<details>` element.
+
+## Scroll timing and element fragment issue
+
+With this feature, find-in-page and ScrollToTextFragment will have an
+asynchronous flow for firing the beforematch event and scrolling when the match
+is in a `content-visibility: hidden-matchable` subtree:
+1. A match within a `content-visibility: hidden-matchable` subtree has been
+   found
+2. Wait for the next requestAnimationFrame
```

Comment:
This should say "Schedule an animation frame, and wait until it happens" or something. It's easy to confuse wait for requestAnimationFrame as in you wait until script or someone calls requestAnimationFrame

Reply:
thanks, you're much better with this terminology than me :)